src/documents/APIChangesFrom2_0To3_0.md

```diff
@@ -121,3 +121,9 @@ All panels must supply a `header` selector, which will be used by the `fluid.pre
         </tr>
     </tbody>
 </table>
+
+## Component API Changes
+
+### Reorderer Component
+
+* The 'stylisticOffset' selector has been removed as it was of no use.
```

Comment:
I think you've used single quotes instead of back ticks around stylisticOffset. Also can you rephrase this to something like. "The `stylisticOffset` selector was no longer in use and has been removed.